autobot/telegram/parser.py

```diff
@@ -697,3 +694,417 @@ def _parse_sticker (self,key:str,val:dict) -> Sticker|StickerSet:
+            passportelementerror_obj = PassportElementErrorFile()
+            for k,v in val.items():
+                setattr(passportelementerror_obj,k,v)
+        elif (key == "passport_element_error_files"):
+            passportelementerror_obj = PassportElementErrorFiles()
+            for k,v in val.items():
+                setattr(passportelementerror_obj,k,v)
+        elif (key == "passport_element_error_front_side"):
+            passportelementerror_obj = PassportElementErrorFrontSide()
+            for k,v in val.items():
+                setattr(passportelementerror_obj,k,v)
+        elif (key == "passport_element_error_reverse_side"):
+            passportelementerror_obj = PassportElementErrorReverseSide()
+            for k,v in val.items():
+                setattr(passportelementerror_obj,k,v)
+        elif (key == "passport_element_error_selfie"):
+            passportelementerror_obj = PassportElementErrorSelfie()
+            for k,v in val.items():
+                setattr(passportelementerror_obj,k,v)
+        elif (key == "passport_element_error_translation_file"):
+            passportelementerror_obj = PassportElementErrorTranslationFile()
+            for k,v in val.items():
+                setattr(passportelementerror_obj,k,v)
+        elif (key == "passport_element_error_translation_files"):
+            passportelementerror_obj = PassportElementErrorTranslationFiles()
+            for k,v in val.items():
+                setattr(passportelementerror_obj,k,v)
+        elif (key == "passport_element_error_unspecified"):
+            passportelementerror_obj = PassportElementErrorUnspecified()
+            for k,v in val.items():
+                setattr(passportelementerror_obj,k,v)
+        return passportelementerror_obj
+
+    def _parse_forumtopic(self,key:str,val:dict) -> ForumTopicCreated|ForumTopicClosed|ForumTopicReopened:
+        forumtopic_obj = None
+        if (key == "forum_topic_created"):
+            forumtopic_obj = ForumTopicCreated()
+            for k,v in val.items():
+                setattr(forumtopic_obj,k,v)
+        elif (key == "forum_topic_closed"):
+            forumtopic_obj = ForumTopicClosed()
+            for k,v in val.items():
+                setattr(forumtopic_obj,k,v)
+        elif (key == "forum_topic_reopened"):
+            forumtopic_obj = ForumTopicReopened()
+            for k,v in val.items():
+                setattr(forumtopic_obj,k,v)
+        return forumtopic_obj
+
+    def _parse_chatmember(self,key:str,val:dict) -> ChatMembers:
+        chatmember_obj = None
+        if (key == "chat_member_updated"):
+            chatmember_obj = ChatMemberUpdated()
+            for k,v in val.items():
+                match k:
+                    case "old_chat_member"|"new_chat_member":
+                        setattr(chatmember_obj,k,self._parse_chatmember(k,v))
+                    case _:
+                        setattr(chatmember_obj,k,v)
```

Comment:
this `match case` statement is not needed here

Reply:
@kallyas  I think I've seen what you're doing here, it skipped my eyes. I think it would be better to stick to the standard way we were doing things for the sake of predictability and easy reviewing. So I guess we can let this one slide.